syntax = "proto3";

package fbsrankings.messages.query;

import "google/protobuf/timestamp.proto";

import "fbsrankings/messages/enums/enums.proto";
import "fbsrankings/messages/options/options.proto";

message CanceledGameResult {
    string game_id = 1;
    string season_id = 2;
    int32 year = 3;
    int32 week = 4;
    google.protobuf.Timestamp date = 5;
    fbsrankings.messages.enums.SeasonSection season_section = 6;
    string home_team_id = 7;
    string home_team_name = 8;
    string away_team_id = 9;
    string away_team_name = 10;
    string notes = 11;
}

message CanceledGamesResult {
    string query_id = 1;
    repeated CanceledGameResult games = 2;
}

message CanceledGamesQuery {
    string query_id = 1;

    option (fbsrankings.messages.options.topic) = "fbsrankings.query.canceled_games";
}
